data/os/Ubuntu/18.04.yaml

```diff
@@ -1,3 +1,4 @@
 ---
 letsencrypt::plugin::dns_rfc2136::package_name: 'python3-certbot-dns-rfc2136'
 letsencrypt::plugin::dns_route53::package_name: 'python3-certbot-dns-route53'
+letsencrypt::plugin::dns_cloudflare::package_name: 'python3-certbot-dns-cloudflare'
```

Comment:
Could you add the same thing for Ubuntu 20.04?

Reply:
Hello @CallumBanbery, Just a quick question can this be used as is with Ubuntu 20.04, or is this not possible at the moment. Or is there a better way to implement dns certbot for cloudflare, mainly looking to do this with a wildcard. 
Thank you in advance.